overtrack_web/templates/navbar.html

```diff
@@ -58,6 +58,9 @@
         </ul>
         <ul class="navbar-nav pull-right">
             {{ navbar_elem('Download Client', 'client') }}
+            <li class="nav-item {{ 'active' if request.path == url_for('faq.faq') else '' }}" style="font-weight: bold;">
```

Comment:
Might be better to extend the `navbar_elem` to allow specifying params (e.g pass through kwargs to the url_for) since otherwise this code is reimplementing the active-route check